package.json

```diff
@@ -41,6 +41,7 @@
     "@types/form-data": "^2.2.1",
     "@types/jest": "^23.3.13",
     "@types/node": "^10.12.18",
+    "colors": "^1.4.0",
```

Comment:
I think we should remove this package. The colors are cool but don't add anything for users